install_config/persistent_storage/topics/glusterfs_support_requirements.adoc

```diff
@@ -0,0 +1,16 @@
+The following requirements must be met to create a supported integration of
+{gluster} and {product-title}.
+
+For {gluster-external} or standalone {gluster}:
+
+- Minimum version: {gluster} 3.1.3
+- All {gluster} nodes must have valid subscriptions to Red Hat Network channels
+and Subscription Manager repositories.
+- {gluster} nodes must adhere to the requirements laid out in the
+link:{gluster-install-link}chap-planning_red_hat_storage_installation[Planning {gluster} Installation].
```

Comment:
Planning {gluster} Installation guide?

Reply:
No, "Planning Red Hat Gluster Storage Installation" is the name of the section being referenced.